report/CommunicationStrategies/combining.tex

```diff
@@ -0,0 +1,33 @@
+\section{Combining the parts}
+Now that we have looked at a couple of methods that has been insparational for us, we will introduce how we believe requirements can be communicated and structured, such that they can automaticly be translated into executable tests and allow the customer to formulate them with ease and with out misunderstandings.
+
+The first part is Design by Contract, we believe that when specifying requirements for a system, they should be formulated as a contract, i.e. precondition, computation, postcondition. 
+Since this forces the customer to think in a bigger context than if we only focus on the computation.
+Also it gives a lot of extra valuable information to the developers, and it allows for converting the requirement into a executable test, see~\autoref{sec:DbC} for the relation between contracts and Hoare Logic.
+
+The second part is how we imagine the customer to create these contracts. 
+But first we need to specify a bit of context.
+Our focus in this project is from the perspective of the customer, the people paying for the system under development.
+We have also choosen to focus on requirements for mobile application development, more specific we have choosen to only focus on development that uses the framework Flutter.
+The reason for this focus is that it narrows the technical aspects to something manageable.
+So when cosnidering Mobile App development from the perspective of a customer, we believe that there are two key aspects that is of interest for the customer.
+The first being, which "screens" are there in the app, by screens we refere to a collection of Graphical User Interaction (GUI) Elements all accessible in a perticular state of the application, and which GUI elements are present on said screens.
```

Comment:
Synes disse to sætninger siger de samme.
Den første siger, at en skærm i appen er en collection af GUI elements, som kan bruges i en given state, mens den anden siger hvilke GUI elementer der er på denne skærm. Skal det være to forskellige ting, eller noget jeg misforstår?

```suggestion
The first being, which "screens" are there in the app, by screens we refere to a collection of Graphical User Interaction (GUI) Elements all accessible in a particular state of the application, and which GUI elements are present on said screens.
```

Reply:
Den siger: Først hvilke skærme systemet skal have og hvilke elementer der skal være på de skærme..

Måske den indskudt sætning som specificerer hvad en skærm er forvirrer ?